changelog.d/9433.misc

```diff
@@ -0,0 +1 @@
+Drop support for ARMv7 in Docker images.
```

Comment:
Should this be a `.docker` or `.removal`?

Reply:
I keep forgetting those things exist

Reply:
I'm putting it in docker, fairly arbitrarily, but removal feels like its more for removing synapse features :shrug: